PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/GraphNodeOrder.java

```diff
@@ -0,0 +1,65 @@
+package nl.tudelft.pl2016gr2.model;
+
+/**
+ * This class is used to decide the order of the nodes in the graph.
+ *
+ * @author Faris
+ */
+public class GraphNodeOrder implements Comparable<GraphNodeOrder> {
```

Comment:
The name of this class is not so clear. It now implies that the class contains a graph (or at least has to do with a graph) while it only applies to a node. Could you rename it to something like NodePosition.